optimum/exporters/openvino/model_patcher.py

```diff
@@ -288,6 +289,73 @@ def __exit__(self, exc_type, exc_value, traceback):
             block.self_attention.core_attention.forward = block.self_attention.core_attention._orig_forward
 
 
+# transformers/models/llama/modeling_llama.py
+# transformers/models/gemma/modeling_gemma.py
+def _update_causal_mask(self, attention_mask, input_tensor, cache_position):
+    if self.config._attn_implementation == "flash_attention_2":
+        if attention_mask is not None and 0.0 in attention_mask:
+            return attention_mask
+        return None
+
+    dtype, device = input_tensor.dtype, input_tensor.device
+    min_dtype = torch.finfo(dtype).min
+
+    # workaround CVS-137270
```

Comment:
I'm not sure that it is good idea to mention openvino jira ticket in external repo

Reply:
removed!